framework/db/Query.php

```diff
@@ -844,6 +849,119 @@ public function addParams($params)
     }
 
     /**
+     * @var array aliases defined for tables used in this query. `tablename => alias`.
```

Comment:
multiple aliases for the same table is quite a rare case and not covered here. This syntax is to allow writing query conditions without the need to care about which alias was defined for a given table. The use cases are queries that are composed in different places over the application, e.g. starting in the relation definition of AR or the default scope, which are later modified in other places of the application.